### ISSUE TITLE
[Feature] Inherit MariaDB labels

### ISSUE THREAD

**Issue by AlcaYezzz** [mariadb-operator/mariadb-operator#82]
**Is your feature request related to a problem? Please describe.**

Our infrastructure rely on some k8s label where we expect specific values :
- app.kubernetes.io/component
- app.kubernetes.io/instance
- app.kubernetes.io/name
- app.kubernetes.io/version

**Describe the solution you'd like**
We'd like to be able to override the label set by mariadb-operator with some custom label.
One way to do this would be to inherit label from mariadb-operator manifest inside child objects (statefulset, pods, ...)

Something like:
```
apiVersion: mariadb.mmontes.io/v1alpha1
kind: MariaDB
metadata:
  name: mariadb
  labels:
    app.kubernetes.io/component: mycomponent
    app.kubernetes.io/instance: myinstance
    app.kubernetes.io/name: myname
    app.kubernetes.io/version: 1.0.0
```

Would create a statefulset and the associated pods with the same labels

Potentially, a configuration, to specify the labels where we seek inheritance could allow backward compatibility.

Something like the configuration done by zalando with their postgres operator:
>inherited_labels list of label keys that can be inherited from the cluster manifest, and added to each child objects (Deployment, StatefulSet, Pod, PVCs, PDB, Service, Endpoints and Secrets) created by the operator. Typical use case is to dynamically pass labels that are specific to a given Postgres cluster, in order to implement NetworkPolicy. The default is empty.

https://opensource.zalando.com/postgres-operator/docs/reference/operator_parameters.html

**Comment by mmontes11** [comment]
Hey @AlcaYezzz !

This is definetly something missing in `mariadb-operator`, it will be implemented as part of the next release. Stay tuned for the `v0.0.12` 🚀 

Contributions are welcome! Take a look at the statefulset and labels builders:
- https://github.com/mariadb-operator/mariadb-operator/blob/main/pkg/builder/statefulset_builder.go#L67..L70
- https://github.com/mariadb-operator/mariadb-operator/blob/main/pkg/builder/labels/labels.go

Thanks for reporting this 🙏🏻

**Comment by mmontes11** [comment]
Hey @AlcaYezzz , this has been implemented in the following PR:
- https://github.com/mariadb-operator/mariadb-operator/pull/94

It will be released in the `v0.0.12`, stay tuned! 🚀

**Comment by mmontes11** [comment]
@Kazyini Yes it has, have you tried this?:
https://github.com/mariadb-operator/mariadb-operator/blob/88fe1fcfc06c8229bde8ba2eca82bad9467cf1b7/examples/manifests/mariadb_v1alpha1_mariadb_metadata.yaml#L7
